docker/Dockerfile

```diff
@@ -16,18 +16,31 @@
 
 ARG PYTHON_VERSION=3.9
 
+FROM docker.io/python:${PYTHON_VERSION}-slim as base
```

Comment:
I'm a bit confused why we have an extra `FROM` statement now?

Reply:
Dealt with in https://github.com/matrix-org/synapse/pull/12385.